### ISSUE TITLE
Date templating not working in version 0.6.0

### ISSUE THREAD

**Issue by danrlyon** [apache/camel-kafka-connector#812]
The date templating appears to be broken in v0.6.0

### 0.5.0
```
camel.sink.endpoint.keyName: ${date:now:yyyyMMdd-HHmmssSSS}-${exchangeId}
```
produces the files names with the date:
```
20201222-182321000-ID-phenocam-connect-connect-6c8fcdd487-t5l6k-1608661396609-0-23
```

### 0.6.0
```
camel.sink.endpoint.keyName: ${date:now:yyyyMMdd-HHmmssSSS}-${exchangeId}
```
produces this key name (keeps overwriting the file):
```
${date:now:yyyyMMdd-HHmmssSSS}-${exchangeId}
```

I'm using the AWS2 S3 Camel Kafka Connector.

**Comment by oscerd** [comment]
This has been fixed in 0.7.0 under vote

**Comment by brokenjacobs** [comment]
Do you have a link to the fix? Didn't see anything in the changelog.

**Comment by oscerd** [comment]
It is a fix coming from camel upstream release 3.7.0, so we are getting it while upgrading.

**Comment by danrlyon** [comment]
Excellent, thanks for the assistance!
